sdcm/cluster.py

```diff
@@ -3429,6 +3429,13 @@ def get_rack_names_per_datacenter_and_rack_idx(self, db_nodes: list[BaseNode] |
 
         return rack_names_mapping
 
+    @staticmethod
+    def get_rack_names_per_datacenter_from_rack_mapping(self, rack_mapping) -> dict[str, list[str]]:
```

Comment:
Why do you have `self` here? It should either be static method without self or a normal method with self, right?

Reply:
Thats exactly how I encountered it, I updated Ruff (PR incoming) and this was one of the checks added by default